docs/docs/reference-react.md

```diff
@@ -0,0 +1,149 @@
+---
+id: reference-react
+title: React Top-Level API
+permalink: docs/reference-react.html
+next: reference-react-dom.html
+redirect_from: "/docs/reference.html"
+---
+
+`React` is the entry point to the React library. If you're using one of the prebuilt packages it's available as a global; if you're using CommonJS modules you can `require()` it.
+
+## Reference
+
+ - [`React.Component`](#react.component)
+ - [`createClass()`](#createclass)
+ - [`createElement()`](#createlement)
+ - [`cloneElement()`](#cloneelement)
+ - [`createFactory()`](#createfactory)
+ - [`isValidElement()`](#isvalidelement)
+ - [`PropTypes`](#proptypes)
+ - [`Children`](#children)
+
+### `React.Component`
+
+```javascript
+class Greeting extends React.Component {
+  render() {
+    return <h1>Hello, {this.props.name}</h1>;
+  }
+}
+```
+
+`React.Component` is the base class for React components when they are defined using ES6 classes. See the [React.Component API Reference](/react/docs/reference-react-component.html) for a list of methods are provided by the base `React.Component` class.
+
+### `createClass()`
+
+```javascript
+React.createClass(specification)
+```
+
+If you don't use ES6 yet, you may use the `React.createClass` helper instead to create a component class. See [Using React without ES6](/react/docs/react-without-es6.html) for more information.
+
+### `createElement()`
+
+```javascript
+React.createElement(
+  type,
+  [props],
+  [children ...]
```

Comment:
The notation in JS is `...children` (it's part of the language). I think we should stick to that.